src/containerapp/azext_containerapp/_validators.py

```diff
@@ -86,3 +89,68 @@ def validate_ingress(namespace):
         if namespace.ingress:
             if not namespace.target_port:
                 raise ValidationError("Usage error: must specify --target-port with --ingress")
+
+
+def _set_ssh_defaults(cmd, namespace):
```

Comment:
NIT: since this technically a validator should the name of the method be like _validate_set (if this doing both)

Reply:
ah! never mind just realized this is a private function & is called inside the validate.